### DISCUSSION TITLE
closed

### DISCUSSION THREAD

**Discussion by b00l1** [aclap-dev/video-downloadhelper#444] [Help and Q&A]

**Comment by jcfrog** [comment]
Hi,
We officially support Edge, Chrome and Firefox.

**Comment by Mattmjs** [comment]
Brave is chromium based so adding the chrome extension shouldn't be a problem. Try the chrome beta extension and see if it helps.

**Comment by GrampaWildWilly** [comment]
The original poster apparently removed his original post & changed the title of this discussion.  This is completely counter to the philosophy of this forum.  Any forum, really.  You ask a question, you get an answer.  You can go away happy.  But maybe somebody else some day might have the same question.  What use is this thread now?  We don't even know what the original question was.  The answers are useful information, but they would be more useful in the context of knowing the question as well.  So I hope folks do NOT get in the habit of doing what this poster did.  Leave your questions as they are.  Don't remove them or even edit them, except to clarify facts or correct spelling or things like that.  Leave the discussion here so it will be useful to other people coming after you.
